dependencies.yaml

```diff
@@ -500,19 +462,68 @@ dependencies:
           - scikit-learn>=0.23.1
       - output_types: [conda]
         packages:
-          - &pylibwholegraph_unsuffixed pylibwholegraph==25.2.*,>=0.0.0a0
           - *thrift
   test_python_pylibcugraph:
     common:
       - output_types: [conda, pyproject]
         packages:
           - *numpy
 
+  depends_on_cugraph:
+    common:
+      - output_types: conda
+        packages:
+          - &cugraph_unsuffixed cugraph==25.2.*,>=0.0.0a0
+      - output_types: requirements
+        packages:
+          # pip recognizes the index as a global option for the requirements.txt file
+          - --extra-index-url=https://pypi.nvidia.com
+          - --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
+    specific:
+      - output_types: [requirements, pyproject]
+        matrices:
+          - matrix:
+              cuda: "12.*"
+              cuda_suffixed: "true"
+            packages:
+              - cugraph-cu12==25.2.*,>=0.0.0a0
+          - matrix:
+              cuda: "11.*"
+              cuda_suffixed: "true"
+            packages:
+              - cugraph-cu11==25.2.*,>=0.0.0a0
+          - {matrix: null, packages: [*cugraph_unsuffixed]}
+
+  depends_on_cugraph_service_client:
+    common:
+      - output_types: conda
+        packages:
+          - &cugraph_service_client_unsuffixed cugraph-service-client==25.2.*,>=0.0.0a0
```

Comment:
This package does not use CUDA or have CUDA dependencies, so it should never be suffixed. This can be simplified.

References:

https://github.com/rapidsai/cugraph/blob/6783aec7a56f05d5088073c822140bc0cd7d9665/python/cugraph-service/client/pyproject.toml#L51

https://anaconda.org/search?q=cugraph-service-client

Reply:
Ah good catch, thanks. Guess the suffixes that were in the file before were unnecessary. I pushed https://github.com/rapidsai/cugraph/pull/4805/commits/7fc94736184cf1fcee3349f650ef8ceecdf0ffc7 removing the suffix-related stuff for that.